docker/rust.docker

```diff
@@ -1,28 +1,38 @@
 FROM codewars/base-runner
 
-# Install Rust Compiler
-RUN curl -sSf https://static.rust-lang.org/rustup.sh | sh
+# Install Rust v1.15.1 to /usr/local
+RUN set -ex \
+ && cd /tmp \
+ && curl -fSsL https://static.rust-lang.org/dist/2017-02-08/rust-1.15.1-x86_64-unknown-linux-gnu.tar.gz -o rust.tar.gz \
```

Comment:
Looks like this would get cargo after all? still though the rustup did seem a little more flexible, looks something like this average version I attempted:

```dockerfile
RUN cd /tmp \
    && curl https://sh.rustup.rs -sSf -o rustup-init.sh \
    && chmod +x rustup-init.sh \
    && sh rustup-init.sh -y --default-toolchain 1.15.1 \
    && rm rustup-init.sh
```